### PR TITLE
P20-1179: Fix language dropdown styles on Lab pages

### PR DESCRIPTION
**Pull request by artem-vavilov**
## Links
- Jira task: [P20-1179](https://codedotorg.atlassian.net/browse/P20-1179)
- Related PR: https://github.com/code-dot-org/code-dot-org/pull/61397

### CODE REVIEW

File: dashboard/test/ui/features/platform/global_edition/region_select.feature [hunk 1]
```diff
@@ -31,23 +31,23 @@ Feature: Global Edition - Region Select
   Scenario: User can switch between the international and regional versions using the language selector on a Lab page
     Given I am on "http://studio.code.org/projects/artist/new"
     And I wait for the lab page to fully load
-    And element "#locale option:checked" contains text "English"
+    And element "#localeForm option:checked" contains text "English"
```
Comment (cearachew) [inline dashboard/test/ui/features/platform/global_edition/region_select.feature:34]: was the id meant to have changed to `localeForm`? it looks like there's no id instead

File: apps/src/sharedComponents/footer/I18nDropdown/index.tsx [hunk 2]
```diff
@@ -32,7 +32,6 @@ const I18nDropdown: React.FC<I18nDropdownProps> = ({
     >
       <input type="hidden" name="user_return_to" value={window.location.href} />
       <SimpleDropdown
-        id="locale"
         className="languageSelect"
```
Comment (wilkie) [inline apps/src/sharedComponents/footer/I18nDropdown/index.tsx:35]:
```suggestion
        id="localeForm"
        className="languageSelect"
```

**Review by cearachew — APPROVED** [review]
:shipit:

**Review by wilkie — APPROVED** [review]

File: dashboard/test/ui/features/platform/global_edition/region_select.feature (hunk 1 above)
Comment (artem-vavilov) [reply dashboard/test/ui/features/platform/global_edition/region_select.feature:34]: https://github.com/code-dot-org/code-dot-org/blob/88877174f9451cdbcc4bb05f562ca56f6ee65c55/apps/src/sharedComponents/footer/I18nDropdown/index.tsx#L30

Comment (artem-vavilov) [reply dashboard/test/ui/features/platform/global_edition/region_select.feature:34]: https://github.com/code-dot-org/code-dot-org/blob/88877174f9451cdbcc4bb05f562ca56f6ee65c55/apps/src/sharedComponents/footer/I18nDropdown/index.tsx#L27-L30

**Pull request merged**